src/utils/parse-date-option.ts

```diff
@@ -0,0 +1,11 @@
+import * as moment from "moment";
+
+const isValidDate = (date: string) => moment(date, "YYYY-MM-DD", true).isValid();
+
+export const parseDateOption = (getTodayDate: () => string, date: string) => {
+    if (date === null || date === undefined || isValidDate(date) === false) {
+        return getTodayDate();
```

Comment:
This would mean that if I pass an invalid date as the `date` option, it would just use today's date, rather than giving an error.

e.g. if I type onetime start --date 30/02/2019, it will just log time today, which is probably not what I'd expect.